### ISSUE TITLE
Error, need to add one of ACCESS_FINE_LOCATION or ACCESS_COARSE_LOCATION to AndroidManifest.xml

### ISSUE THREAD

**Issue by wil93** [loup-v/geolocation#36]
I'm getting this error:

    "Missing location permission in AndroidManifest.xml. You need to add one of ACCESS_FINE_LOCATION or ACCESS_COARSE_LOCATION. See readme for details."

However, I already added ACCESS_COARSE_LOCATION:

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.example.ranky">

    <!-- The INTERNET permission is required for development. Specifically,
         flutter needs it to communicate with the running application
         to allow setting breakpoints, to provide hot reload, etc.
    -->
    <uses-permission android:name="android.permission.INTERNET"/>

    <!-- The location permission is required for showing close contests to
         the user.
    -->
    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />

...
```

**Comment by nosmirck** [comment]
I'm having the exact same issue. Any leads?

**Comment by lukaspili** [comment]
Sorry for the delay. It should be fixed in the latest release 1.0.0.
